dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/scheduling/JobScheduler.java

```diff
@@ -221,6 +221,7 @@ private void runDueJob(JobConfiguration config, Instant start) {
               jobId, start.atZone(ZoneId.systemDefault())));
       return;
     }
+    log.debug("Running job %s");
```

Comment:
@jbee this shows up as `Running job %s` in the logs I assume this should be `log.debug("Running job {}", jobId);`